src/shared/prompts/ja/parts/structured_json_schema_instruction.md

```
@@ -5,8 +5,8 @@
 -->
 {{instruction}}
 
-Return exactly one fenced JSON block that matches this JSON schema:
+次の JSON schema に一致する fenced JSON block をちょうど1つ返してください:
 ```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)